Funcky/Extensions/EnumerableExtensions/GetNonEnumeratedCountOrNone.cs

```diff
@@ -0,0 +1,11 @@
+namespace Funcky.Extensions;
+
+public static partial class EnumerableExtensions
+{
+#if TRY_GET_NON_ENUMERATED_COUNT
+    public static Option<int> GetNonEnumeratedCountOrNone<TSource>(this IEnumerable<TSource> source)
+        => source.TryGetNonEnumeratedCount(out var count)
+            ? count
+            : Option.None<int>();
```

Comment:
```suggestion
            : Option<int>.None;
```

Reply:
Fixed in 91c13a21e8bb89833140655bfb0c608ad7368c31